### DISCUSSION TITLE
Integrate with DataDog RUM

### DISCUSSION THREAD

**Discussion by azisso** [GitbookIO/community#250] [Feature Requests]
We are used to sending user sessions and activity into [RUM](https://www.datadoghq.com/product/real-user-monitoring/) to track and measure metrics. Is it possible to have Gitbook integrate with it?

**Comment by liranmarkin** [comment]
Upvoting! Since Datadog has released the Real User Monitoring we also are using this, it provides us a one platform for all our monitoring activities. Will be glad to integrate gitbook with it.
